### PR TITLE
Release Hermes v1.1.0

### PR DESCRIPTION
**Pull request by romac**
Closes: #2738

⚡

- [x] Create a new release in the changelog, using [`unclog`](https://github.com/informalsystems/unclog).
  - If doing a release candidate (`rc`) version, then skip the `unclog release` step.
- [x] Bump all crate versions to the new version.
- [x] Reassign unfinished issues of previous milestone to the next milestone.
- [x] Update Cargo.lock file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,128 @@
...
+  ([#2677](https://github.com/informalsystems/ibc-rs/issues/2677))
+- Remove the `ibc-proto` and `ibc-proto-compiler` crates from the repository
+  ([#2667](https://github.com/informalsystems/ibc-rs/issues/2667))
+
+#### BUG FIXES
+
+- Fix incorrect Github workflow trigger paths
+  ([#2667](https://github.com/informalsystems/ibc-rs/issues/2667))
+- Fix comment in `config.toml` to correctly state that the default value of `clear_on_start` is `true`
+  ([#2750](https://github.com/informalsystems/ibc-rs/issues/2750))
```
Comment (ljoss17) [inline CHANGELOG.md:44]:
```suggestion
  ([#2750](https://github.com/informalsystems/hermes/issues/2750))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,128 @@
...
+
+
+### Hermes - `ibc-relayer-cli` (v1.1.0)
+
+#### FEATURES
+
+- Add `hermes auto config` CLI command to automatically generate a config file with data from the chain registry
+  ([#2187](https://github.com/informalsystems/ibc-rs/issues/2187))
```
Comment (ljoss17) [inline CHANGELOG.md:52]:
```suggestion
  ([#2187](https://github.com/informalsystems/hermes/issues/2187))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,128 @@
...
+- Added new optional flag `--packet-data-query-height` to CLI `hermes tx packet-recv`
+  in order to specify the height at which the packet data is queried
+  ([#2453](https://github.com/informalsystems/ibc-rs/issues/2453))
```
Comment (ljoss17) [inline CHANGELOG.md:55]:
```suggestion
  ([#2453](https://github.com/informalsystems/hermes/issues/2453))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,128 @@
...
+- New command `fee transfer` which transfers tokens with fees
+  ([#2714](https://github.com/informalsystems/ibc-rs/issues/2714))
```
Comment (ljoss17) [inline CHANGELOG.md:57]:
```suggestion
  ([#2714](https://github.com/informalsystems/hermes/issues/2714))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,128 @@
...
+
+- Added Evmos compatible integration tests
+  ([#2442](https://github.com/informalsystems/ibc-rs/issues/2442))
+
+### Guide
+
+#### IMPROVEMENTS
+
+- Integration with `mdbook` templates
+  ([#2605](https://github.com/informalsystems/ibc-rs/issues/2605))
```
Comment (ljoss17) [inline CHANGELOG.md:121]:
```suggestion
  ([#2605](https://github.com/informalsystems/hermes/issues/2605))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,128 @@
...
+  ([#2187](https://github.com/informalsystems/ibc-rs/issues/2187))
+
+[cosmos-chain-registry]: https://github.com/cosmos/chain-registry
+
+### `ibc-test-framework` (v0.20.0)
+
+#### FEATURES
+
+- Added Evmos compatible integration tests
+  ([#2442](https://github.com/informalsystems/ibc-rs/issues/2442))
```
Comment (ljoss17) [inline CHANGELOG.md:114]:
```suggestion
  ([#2442](https://github.com/informalsystems/hermes/issues/2442))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,128 @@
...
+  and improve client refresh frequency to depend on the `trusting_period`
+  ([#2487](https://github.com/informalsystems/ibc-rs/issues/2487))
+- Set `max_tx_size` default value to 180KB instead of 2MB
+  ([#2595](https://github.com/informalsystems/ibc-rs/issues/2595))
+
+
+### Chain Registry - `ibc-chain-registry` (v0.20.0)
+
+- New crate `ibc-chain-registry` to fetch data from the [chain-registry][cosmos-chain-registry] and query RPC/gRPC endpoints
+  ([#2187](https://github.com/informalsystems/ibc-rs/issues/2187))
```
Comment (ljoss17) [inline CHANGELOG.md:105]:
```suggestion
  ([#2187](https://github.com/informalsystems/hermes/issues/2187))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,128 @@
...
+
+- Bump compatibility with IBC-Go v5 and Cosmos SDK v0.46
+  ([#2532](https://github.com/informalsystems/ibc-rs/issues/2532))
+- Account for full transaction encoding when batching messages
+  ([#2575](https://github.com/informalsystems/ibc-rs/issues/2575))
+- Add a health check to warn user if `gas_multiplier` is smaller than 1.1,
+  and improve client refresh frequency to depend on the `trusting_period`
+  ([#2487](https://github.com/informalsystems/ibc-rs/issues/2487))
+- Set `max_tx_size` default value to 180KB instead of 2MB
+  ([#2595](https://github.com/informalsystems/ibc-rs/issues/2595))
```
Comment (ljoss17) [inline CHANGELOG.md:99]:
```suggestion
  ([#2595](https://github.com/informalsystems/hermes/issues/2595))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,128 @@
...
+
+#### IMPROVEMENTS
+
+- Bump compatibility with IBC-Go v5 and Cosmos SDK v0.46
+  ([#2532](https://github.com/informalsystems/ibc-rs/issues/2532))
+- Account for full transaction encoding when batching messages
+  ([#2575](https://github.com/informalsystems/ibc-rs/issues/2575))
+- Add a health check to warn user if `gas_multiplier` is smaller than 1.1,
+  and improve client refresh frequency to depend on the `trusting_period`
+  ([#2487](https://github.com/informalsystems/ibc-rs/issues/2487))
```
Comment (ljoss17) [inline CHANGELOG.md:97]:
```suggestion
  ([#2487](https://github.com/informalsystems/hermes/issues/2487))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,128 @@
...
+
+- Support custom extension options to be able to specify `max_priority_price` for Ethermint dynamic tx fee
+  ([#2566](https://github.com/informalsystems/ibc-rs/issues/2566))
+
+#### IMPROVEMENTS
+
+- Bump compatibility with IBC-Go v5 and Cosmos SDK v0.46
+  ([#2532](https://github.com/informalsystems/ibc-rs/issues/2532))
+- Account for full transaction encoding when batching messages
+  ([#2575](https://github.com/informalsystems/ibc-rs/issues/2575))
```
Comment (ljoss17) [inline CHANGELOG.md:94]:
```suggestion
  ([#2575](https://github.com/informalsystems/hermes/pull/2575))
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,128 @@
...
+
+#### FEATURES
+
+- Support custom extension options to be able to specify `max_priority_price` for Ethermint dynamic tx fee
+  ([#2566](https://github.com/informalsystems/ibc-rs/issues/2566))
+
+#### IMPROVEMENTS
+
+- Bump compatibility with IBC-Go v5 and Cosmos SDK v0.46
+  ([#2532](https://github.com/informalsystems/ibc-rs/issues/2532))
```
Comment (ljoss17) [inline CHANGELOG.md:92]:
```suggestion
  ([#2532](https://github.com/informalsystems/hermes/issues/2532))
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,128 @@
...
+- Fix several bugs which were preventing Hermes to clear packets on ordered channels
+  in some specific conditions, as exhibited on the Interchain Security testnet
+  ([#2670](https://github.com/informalsystems/ibc-rs/issues/2670))
+- Fix how headers are decoded from events
+  ([#2739](https://github.com/informalsystems/ibc-rs/issues/2739))
+
+#### FEATURES
+
+- Support custom extension options to be able to specify `max_priority_price` for Ethermint dynamic tx fee
+  ([#2566](https://github.com/informalsystems/ibc-rs/issues/2566))
```
Comment (ljoss17) [inline CHANGELOG.md:87]:
```suggestion
  ([#2566](https://github.com/informalsystems/hermes/issues/2566))
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,128 @@
...
+
+#### BUG FIXES
+
+- The channel and connection workers do not act needlessly on `NewBlock` events
+  anymore ([#2484](https://github.com/informalsystems/ibc-rs/issues/2484))
+- Fix several bugs which were preventing Hermes to clear packets on ordered channels
+  in some specific conditions, as exhibited on the Interchain Security testnet
+  ([#2670](https://github.com/informalsystems/ibc-rs/issues/2670))
+- Fix how headers are decoded from events
+  ([#2739](https://github.com/informalsystems/ibc-rs/issues/2739))
```
Comment (ljoss17) [inline CHANGELOG.md:82]:
```suggestion
  ([#2739](https://github.com/informalsystems/hermes/issues/2739))
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,128 @@
...
+
+### Relayer Library - `ibc-relayer` (v0.20.0)
+
+#### BUG FIXES
+
+- The channel and connection workers do not act needlessly on `NewBlock` events
+  anymore ([#2484](https://github.com/informalsystems/ibc-rs/issues/2484))
+- Fix several bugs which were preventing Hermes to clear packets on ordered channels
+  in some specific conditions, as exhibited on the Interchain Security testnet
+  ([#2670](https://github.com/informalsystems/ibc-rs/issues/2670))
```
Comment (ljoss17) [inline CHANGELOG.md:80]:
```suggestion
  ([#2670](https://github.com/informalsystems/hermes/issues/2670))
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,128 @@
...
+  Specify the `--all` flag to get the balance for all denominations
+  ([#2726](https://github.com/informalsystems/ibc-rs/issues/2726))
+
+
+### Relayer Library - `ibc-relayer` (v0.20.0)
+
+#### BUG FIXES
+
+- The channel and connection workers do not act needlessly on `NewBlock` events
+  anymore ([#2484](https://github.com/informalsystems/ibc-rs/issues/2484))
```
Comment (ljoss17) [inline CHANGELOG.md:77]:
```suggestion
  anymore ([#2484](https://github.com/informalsystems/hermes/issues/2484))
```

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,5 +1,128 @@
...
+
+- Clean up the logs emitted by the relayer and add more
+  structured information to the messages recorded in the logs
+  ([#1538](https://github.com/informalsystems/ibc-rs/issues/1538))
+- Log the packets cleared by the `clear packets` command
+  ([#2653](https://github.com/informalsystems/ibc-rs/issues/2653))
+- Add a new optional flag, `--denom` to the `hermes keys balance` command in order
+  to specify for which denomination the balance is queried.
+  Specify the `--all` flag to get the balance for all denominations
+  ([#2726](https://github.com/informalsystems/ibc-rs/issues/2726))
```
Comment (ljoss17) [inline CHANGELOG.md:69]:
```suggestion
  ([#2726](https://github.com/informalsystems/hermes/issues/2726))
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,5 +1,128 @@
...
+- New command `fee transfer` which transfers tokens with fees
+  ([#2714](https://github.com/informalsystems/ibc-rs/issues/2714))
+
+#### IMPROVEMENTS
+
+- Clean up the logs emitted by the relayer and add more
+  structured information to the messages recorded in the logs
+  ([#1538](https://github.com/informalsystems/ibc-rs/issues/1538))
+- Log the packets cleared by the `clear packets` command
+  ([#2653](https://github.com/informalsystems/ibc-rs/issues/2653))
```
Comment (ljoss17) [inline CHANGELOG.md:65]:
```suggestion
  ([#2653](https://github.com/informalsystems/hermes/pull/2653))
```

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,5 +1,128 @@
...
+  in order to specify the height at which the packet data is queried
+  ([#2453](https://github.com/informalsystems/ibc-rs/issues/2453))
+- New command `fee transfer` which transfers tokens with fees
+  ([#2714](https://github.com/informalsystems/ibc-rs/issues/2714))
+
+#### IMPROVEMENTS
+
+- Clean up the logs emitted by the relayer and add more
+  structured information to the messages recorded in the logs
+  ([#1538](https://github.com/informalsystems/ibc-rs/issues/1538))
```
Comment (ljoss17) [inline CHANGELOG.md:63]:
```suggestion
  ([#1538](https://github.com/informalsystems/hermes/issues/1538))
```

**Review by ljoss17 — COMMENTED** [review]
Urls are still referring to previous repo

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,5 +1,128 @@
...
+
+[ibc]: https://github.com/cosmos/ibc-rs
+[ibc-proto]: https://github.com/cosmos/ibc-proto-rs
+[cosmos]: https://github.com/cosmos
+[split-tweet]: https://twitter.com/informalinc/status/1578120684508692481
+[split-issue]: https://github.com/informalsystems/hermes/issues/2639
+
+### General
+
+
```
Comment (seanchen1991) [inline CHANGELOG.md:31]:
```suggestion
The `gm` binary has been split out of the repo and has been moved into its own [repository](https://github.com/informalsystems/gm). This change doesn't affect current local instances of the `gm` binary, though new versions will need to be sourced from the new repository by running `git clone informalsystems/gm` and running the install script from there. 

```

**Review by seanchen1991 — APPROVED** [review]
Added a suggestion to the changelog summary regarding `gm` being split out. Looks good to me otherwise.

**Review by ljoss17 — APPROVED** [review]

**Pull request merged**